lib/mixpanel-node.js

```diff
@@ -126,13 +126,16 @@ var create_client = function(token, config) {
                                         finished or an error occurs
     */
     metrics.track = function(event, properties, callback) {
+        var now = new Date(),
+            importTimeLimit = parse_time(now.setDate(now.getDate() - 5));
```

Comment:
Could you change these vars from camelCase to under_scored? (Not my favorite either, but since that's the style in the rest of the lib we'd like to keep it internally consistent.)